website/promotion/migrations/0006_auto_20211215_1201.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.9 on 2021-12-15 11:01
+
+import datetime
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('events', '0049_event_optional_registrations'),
+        ('promotion', '0005_auto_20211215_1140'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='promotionrequest',
+            name='channel',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='promotion.promotionchannel', verbose_name='channel'),
+        ),
+        migrations.AlterField(
+            model_name='promotionrequest',
+            name='event',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='events.event', verbose_name='event'),
+        ),
+        migrations.AlterField(
+            model_name='promotionrequest',
+            name='publish_date',
+            field=models.DateField(default=datetime.date.today, verbose_name='Publish date'),
+        ),
+        migrations.AlterField(
+            model_name='promotionrequest',
+            name='status',
+            field=models.CharField(choices=[('not_started', 'Not started'), ('started', 'Started'), ('finished', 'Finished'), ('published', 'Published')], default='not_started', max_length=40, verbose_name='status'),
+        ),
+    ]
```

Comment:
When all model changes are done, please merge all migrations (remove all of them, re-make them)